app/lib/client/fetcher.dart

```diff
@@ -99,21 +101,32 @@ class MeinUnterrichtFetcher extends Fetcher<Lessons> {
   }
 }
 
-class VisibleConversationsFetcher extends Fetcher<dynamic> {
-  VisibleConversationsFetcher(super.validCacheDuration, {super.storageKey});
+class ConversationsFetcher extends Fetcher<List<OverviewEntry>> {
+  ConversationsFetcher(super.validCacheDuration, {super.storageKey});
+  bool _suspend = false;
 
   @override
-  Future<dynamic> _get() {
-    return client.conversations.getOverview(false);
+  void _timerCallback(Timer timer) {
+    if (!_suspend) {
+      super._timerCallback(timer);
+    }
   }
-}
-
-class InvisibleConversationsFetcher extends Fetcher<dynamic> {
-  InvisibleConversationsFetcher(super.validCacheDuration, {super.storageKey});
 
   @override
-  Future<dynamic> _get() {
-    return client.conversations.getOverview(true);
+  Future<List<OverviewEntry>> _get() {
+    return client.conversations.getOverview();
+  }
+
+  void toggleSuspend() {
+    _suspend = !_suspend;
+  }
+
+  /// Force pushes a new supply for the stream.
+  void supply(final List<OverviewEntry> content) {
```

Comment:
I get what this function does, but I do not really get the name "Angebot" / "liefern" ?